src/cloudant/design_document.py

```diff
@@ -79,6 +89,22 @@ def add_view(self, view_name, map_func, reduce_func=None, **kwargs):
         view = View(self, view_name, map_func, reduce_func, **kwargs)
         self.views.__setitem__(view_name, view)
 
+    def add_search_index(self, search_index_name, search_index, **kwargs):
+        """
+        Appends a Cloudant search index to the locally cached DesignDocument
+        SearchIndex dictionary.
+
+        :param str search_index_name: Name used to identify the SearchIndex.
+        :param str search_index: Javascript search index function.
+        """
+        if self.get_search_index(search_index_name) is not None:
+            msg = "Search index {0} already exists in this design doc"\
```

Comment:
I think we touched upon this some time before, we are not using the `\` notation anywhere else in the this library.  Consider breaking this `msg` assignment up differently.  maybe on the open parens in the `format`.

This comment applies to the entire PR.